### PR TITLE
Support wider styling set for fixed layer. Re-enable forceTransfer on sidebar

### LINKED ISSUE #6699: Support wider styling set for fixed layer. Re-enable forceTransfer on sidebar.
See https://www.google.com/amp/s/wsbtv.relaymedia.com/amp/news/local/south-fulton-county/closing-arguments-begin-in-taser-death-trial/476224280#amph=1

Issues happen due to styles, such as `body#CD > abc`. We need to see if we can also copy "ID" from BODY to the fixed layer.

### PR DESCRIPTION
**Pull request by camelburrito**
Fixes #6699

### CODE REVIEW

File: extensions/amp-sidebar/0.1/amp-sidebar.js [hunk 1]
```diff
@@ -85,7 +85,7 @@ export class AmpSidebar extends AMP.BaseElement {
 
     // TODO(camelburrito, #6699): Return forceTransfer=false once fixed layer
```
Comment (dvoytenko) [inline extensions/amp-sidebar/0.1/amp-sidebar.js]: Remove TODO

File: src/service/fixed-layer.js [hunk 2]
```diff
@@ -607,8 +607,10 @@ export class FixedLayer {
     if (!this.transfer_ || this.fixedLayer_) {
       return this.fixedLayer_;
     }
-    this.fixedLayer_ = this.ampdoc.win.document.createElement('body');
-    this.fixedLayer_.id = '-amp-fixedlayer';
+    const doc = this.ampdoc.win.document;
+    const bodyId_ = doc.getElementsByTagName('body')[0].getAttribute('id');
```
Comment (dvoytenko) [inline src/service/fixed-layer.js]: Remove `_`. It should be instead `const bodyId = doc.body.id;`

File: src/service/fixed-layer.js [hunk 3]
```diff
@@ -607,8 +607,10 @@ export class FixedLayer {
...
+    this.fixedLayer_ = doc.createElement('body');
+    this.fixedLayer_.id = bodyId_ || '-amp-fixedlayer';
```
Comment (dvoytenko) [inline src/service/fixed-layer.js]: And, yes, definitely tests :)

File: src/service/fixed-layer.js (hunk 2 above)
Comment (camelburrito) [inline src/service/fixed-layer.js]: done

**Comment by camelburrito** [comment]
PTAL

File: src/service/fixed-layer.js [hunk 4]
```diff
@@ -607,8 +607,10 @@ export class FixedLayer {
     if (!this.transfer_ || this.fixedLayer_) {
       return this.fixedLayer_;
     }
-    this.fixedLayer_ = this.ampdoc.win.document.createElement('body');
-    this.fixedLayer_.id = '-amp-fixedlayer';
+    const doc = this.ampdoc.win.document;
+    const bodyId = doc.body.id;
+    this.fixedLayer_ = doc.createElement('body');
```
Comment (jridgewell) [inline src/service/fixed-layer.js]: Why not do a shallow clone here? Then we have `class`, `id` and any other attributes.

Comment (dvoytenko) [inline src/service/fixed-layer.js]: Seems like shallow clone would work great, yes.

Comment (dvoytenko) [inline src/service/fixed-layer.js]: But we'll have to reset `fixedLayer.style` because we override it for real body for many things.

**Pull request merged**